homeassistant/components/systemmonitor/coordinator.py

```diff
@@ -113,21 +125,28 @@ class SystemMonitorNetIOCoordinator(MonitorCoordinator[dict[str, snetio]]):
 
     def update_data(self) -> dict[str, snetio]:
         """Fetch data."""
-        return psutil.net_io_counters(pernic=True)
+        io_counters: dict[str, snetio] = self._psutil.net_io_counters(pernic=True)
+        return io_counters
 
 
 class SystemMonitorNetAddrCoordinator(MonitorCoordinator[dict[str, list[snicaddr]]]):
     """A System monitor Network Address Data Update Coordinator."""
 
     def update_data(self) -> dict[str, list[snicaddr]]:
         """Fetch data."""
-        return psutil.net_if_addrs()
+        addresses: dict[str, list[snicaddr]] = self._psutil.net_if_addrs()
+        return addresses
 
 
-class SystemMonitorLoadCoordinator(MonitorCoordinator[tuple[float, float, float]]):
+class SystemMonitorLoadCoordinator(
+    MonitorCoordinator[tuple[float, float, float] | None]
+):
     """A System monitor Load Data Update Coordinator."""
 
-    def update_data(self) -> tuple[float, float, float]:
+    def update_data(self) -> tuple[float, float, float] | None:
+        """Coordinator is not async."""
+
+    async def _async_update_data(self) -> tuple[float, float, float] | None:
         """Fetch data."""
         return os.getloadavg()
```

Comment:
Verified this is a non-blocking syscall so good to run in the in the event loop
`sysinfo({uptime=1483159, loads=[2976, 4288, 5344], totalram=16389873664, freeram=3412480000, sharedram=1642496, bufferram=432185344, totalswap=5408763904, freeswap=5408763904, procs=693, totalhigh=0, freehigh=0, mem_unit=1}) = 0`